### DISCUSSION TITLE
Docker离线下载的映射问题

### DISCUSSION THREAD

**Discussion by Memorywzd** [AlistGo/alist#4097] [General]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [ ] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

v3.15.1

### Driver used / 使用的存储驱动

none

### Describe the bug / 问题描述

关于离线下载，文档原文摘抄如下：
>如果你是 docker，请自行映射以下两个默认路径
>
>- /opt/alist/data/temp/aria2
>
>- /opt/alist/data/temp/qbittorrent

一般情况下，此处应该被理解为”将临时文件保存至映射得到的目录“，因此本人将其映射至了nas中的下载文件夹。然而，docker运行后，报错如下：
`ERRO[2023-04-09 07:22:37] failed delete temp file: unlinkat /opt/alist/data/temp/aria2: device or resource busy`  

检测下载文件夹发现，文件已被删除一空。  

请问，为何文档不提前声明该目录会在运行时被清空，以及为什么要在启动时清空”未经详细描述“的”缓存目录“？

### Reproduction / 复现链接

none

### Logs / 日志

INFO[2023-04-09 07:22:37] reading config file: data/config.json        
INFO[2023-04-09 07:22:37] load config from env with prefix:            
ERRO[2023-04-09 07:22:37] failed delete temp file: unlinkat /opt/alist/data/temp/aria2: device or resource busy 
INFO[2023-04-09 07:22:37] init logrus...                               
INFO[2023-04-09 07:22:37] start server @ 0.0.0.0:5244                  
INFO[2023-04-09 07:22:37] qbittorrent not ready.

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xhofe** [comment]
temp为alist独占的临时文件夹，为避免程序中断产生垃圾文件故删除。

**Comment by Memorywzd** [comment]
请删除文档中的错误描述，把本解释更新至文档中。删掉关于映射文件夹的描述，防止他人文件再受到损失

**Comment by ababtion** [comment]
所以说应该怎么解决呢？
